FrontMen/fm-hours
# Feature/employee month report

# Changes
New button on week view lets you navigate to monthly overview

## Related issues
https://github.com/FrontMen/fm-hours/issues/151

## Added
Weekly overview page:
* can filter by billable or all
* can filter by customer
* can navigate back to weekly view
* can trigger a brwoser print of the selected records
* can change the month to be viewed, up until the current month

## Changed
New button on week view lets you navigate to monthly overview

## How to test
1. go to week view
2. click on monthly overview - shows the current month, not the month corresponding to the week you came from
3. filter the overview by billable only - it should show only billables in the table
4. filter by project - it should only show that project. Total time for that project should match the available entries
5. change month - data in table is updated to correctly reflect the selected month
6. go to present - should show you the entire present month
7. print button - should trigger window.print, the layout should be print-friendly, with elements that only appear in print
8. Navigate back to the "/" - shows the current week, not the week you came from

## Screenshots
![image](https://user-images.githubusercontent.com/85111889/125920042-6c74a8b9-bd58-43b1-a8c1-7a1cac239e62.png)
![image](https://user-images.githubusercontent.com/85111889/125921548-ebdf6555-93c2-4e6b-b16c-bb500b7ef40b.png)
![image](https://user-images.githubusercontent.com/85111889/125921592-2fa5cabc-8c25-4d85-8ed5-ed70b7685a0e.png)
